docs/source/openvino/export.mdx

```diff
@@ -78,7 +78,8 @@ Optional arguments:
   --ratio RATIO         A parameter used when applying 4-bit quantization to control the ratio between 4-bit and 8-bit
                         quantization. If set to 0.8, 80% of the layers will be quantized to int4 while 20% will be
                         quantized to int8. This helps to achieve better accuracy at the sacrifice of the model size
-                        and inference latency. Default value is 1.0.
+                        and inference latency. Default value is 1.0. Note: If dataset is provided, and the ratio is
+                        less than 1.0, then data-aware mixed precision assignment will be applied.
   --sym                 Whether to apply symmetric quantization
```

Comment:
What do you think about adding a warning when a dataset is provided while the quantization methodology is default and `ratio=1` to specify that the dataset will be ignored in this case?

Reply:
Thanks for the idea. Added.

Reply:
thanks a lot @nikita-savelyevv